addons/mail/static/src/models/discuss/discuss.js

```diff
@@ -144,6 +64,11 @@ registerModel({
             }
             this.threadView.topbar.openInvitePopoverView();
         },
+        open() {
```

Comment:
This method name is very confusing because it doesn't actually open the discuss app, it merely creates the view when it is missing, which is just technical reason.

Reply:
We had the boolean field `isOpen` and opening/closing discuss essentially came from updating this boolean field.
Now that it's the presence of the view that determine that, and also we have special setup for view creation (i.e. some `doFocus`), made sense to abstract implementation detail with functions `open/close`.

I don't think this is misleading, this really opens discuss. If the view is already there, it means discuss is already open so it does nothing, but discuss is open, so it follows intention of desire to open discuss.